monitoring/ceph-mixin/dashboards/host.libsonnet

```diff
@@ -1,779 +1,794 @@
-        ) + { gridPos: { x: x, y: y, w: w, h: h } };
-
-      local HostsOverviewGraphPanel(title, description, formatY1, expr, legendFormat, x, y, w, h) =
-        u.graphPanelSchema(
-          {}, title, description, 'null', false, formatY1, 'short', null, null, 0, 1, '$datasource'
-        )
-        .addTargets(
-          [u.addTargetSchema(
-            expr, legendFormat
-          )]
-        ) + { gridPos: { x: x, y: y, w: w, h: h } };
-
-      u.dashboardSchema(
-        'Host Overview',
-        '',
-        'y0KGL0iZz',
-        'now-1h',
-        '10s',
-        16,
-        [],
-        '',
-        {
-          refresh_intervals: ['5s', '10s', '30s', '1m', '5m', '15m', '30m', '1h', '2h', '1d'],
-          time_options: ['5m', '15m', '1h', '6h', '12h', '24h', '2d', '7d', '30d'],
-        }
-      )
-      .addRequired(
-        type='grafana', id='grafana', name='Grafana', version='5.3.2'
-      )
-      .addRequired(
-        type='panel', id='graph', name='Graph', version='5.0.0'
-      )
-      .addRequired(
-        type='panel', id='singlestat', name='Singlestat', version='5.0.0'
-      )
-      .addAnnotation(
-        u.addAnnotationSchema(
-          1,
-          '-- Grafana --',
-          true,
-          true,
-          'rgba(0, 211, 255, 1)',
-          'Annotations & Alerts',
-          'dashboard'
-        )
-      )
-      .addTemplate(
-        g.template.datasource('datasource',
-                              'prometheus',
-                              'default',
-                              label='Data Source')
-      )
-      .addTemplate(
-        u.addTemplateSchema('osd_hosts',
-                            '$datasource',
-                            'label_values(ceph_disk_occupation, exported_instance)',
-                            1,
-                            true,
-                            1,
-                            null,
-                            '([^.]*).*')
-      )
-      .addTemplate(
-        u.addTemplateSchema('mon_hosts',
-                            '$datasource',
-                            'label_values(ceph_mon_metadata, ceph_daemon)',
-                            1,
-                            true,
-                            1,
-                            null,
-                            'mon.(.*)')
-      )
-      .addTemplate(
-        u.addTemplateSchema('mds_hosts',
+(import 'utils.libsonnet') {
+  'hosts-overview.json':
+    local HostsOverviewSingleStatPanel(format,
+                                       title,
+                                       description,
+                                       valueName,
+                                       expr,
+                                       instant,
+                                       x,
+                                       y,
+                                       w,
+                                       h) =
+      $.addSingleStatSchema(['#299c46', 'rgba(237, 129, 40, 0.89)', '#d44a3a'],
```

Comment:
More color literals over here. I cannot tell whether those were intentional or the Grafana defaults... 🤔 if we could somehow replace them with 'color_1_fg', 'color_1_bg', ... or something similar...